docs/configuration/configuration.md

```diff
@@ -29,6 +29,8 @@ An example [oauth2-proxy.cfg]({{ site.gitweb }}/contrib/oauth2-proxy.cfg.example
 | `-authenticated-emails-file` | string | authenticate against emails via file (one per line) | |
 | `-azure-tenant string` | string | go to a tenant-specific or common (tenant-independent) endpoint. | `"common"` |
 | `-basic-auth-password` | string | the password to set when passing the HTTP Basic Auth header | |
+| `-claim-authorization` | string | define a claim-based authorization rule (may be repeated) | |
```

Comment:
For upstream we have:
> ```
> | `-upstream` | string \| list | the http url(s) of the upstream endpoint, file:// paths for static files or `static://<status_code>` for static response. Routing is based on the path | |	
> ```

Rather than saying `(may be repeated)` we should be consistant:
```
| `-claim-authorization` | string\|list | define a claim-based authorization rule(s) | |
```

Reply:
This doc is out of date in this case. It's singular now due to how options parsing splits on commas. Will tidy this up. Thanks for catching it.